src/ResourceManager/ServiceFabric/Commands.ServiceFabric/Commands/ServiceFabricClusterCertificateCmdlet.cs

```diff
@@ -586,7 +594,16 @@ private string GetThumbprintFromSecret(string secretUrl)
                 throw new PSArgumentException("secretUrl");
             }
 
-            var secretBundle = this.KeyVaultClient.GetSecretAsync(secretUrl).Result;
+            SecretBundle secretBundle;
+            try
+            {
+                secretBundle = this.KeyVaultClient.GetSecretAsync(secretUrl).Result;
+            }
+            catch (Exception ex)
+            {
+                throw GetInnerException(ex);
```

Comment:
Any particular reason for this? This usage replaces the callstack of original exception and should be avoided unless there's a strong reason.

Reply:
Not doing this was the cause of this issue: Azure/service-fabric-issues#1054 where people had to see the exception with $error[0].Exception.InnerExceptions